mvp-spec/README.md

```diff
@@ -14,13 +14,13 @@ flowchart LR
     
     Participant("Participant website")
     
-    Participant -->|read user ids & preferences| O
+    Participant -->|read User Id & Preferences| O
```

Comment:
To be honest, I've been confused by the usage of capital letters in documents.
Are we introducing the rule that any word that is part of the glossary should be with a capital first letter?
I'm ok with that, but I'd like this to be consistent across the documents: in my documents so far I tend to _never_ use capitals, in other documents I have the feeling there are capital letters a little more than just for "PAF main definitions".
WDYT?

Reply:
Indeed, that was the convention that I was introducing (without mentioning it anywhere so far :) ) :

Words starting with a capital letter refer to PAF specific terms and must be defined in the glossary (now part of "workflows" file). These should be the only words starting with a capital letter in the repo, and they should always start with a capital letter.

(SSP and DSP, as acronyms, are in capital letters but do not need to be defined in the glossary).

WDYT?

Reply:
I'm ok with that. We'd have to use this convention all over the project (and only for relevant terms).

Again, for consistency, we should agree on the names: I tend to use Id as a shorter name, but I think @RomainLofaso was more carefully using "Identifier". Not a big deal, but if we care about PAF terms, I think it's worth being consistent on this particular one as well.

Reply:
Yes, the term I prefer is "User Id and Preferences", and I'm also removing references to PAF Data, as it is less explicit.

Is that OK for you?

Reply:
I guess we need to add a doc for the glossary so that we are aligned across all the documents.
As a starter, we can extract the one that is at the top of the Audit Requirements. I prefer "User Id and Preferences". Intuitively (without the glossary), PAF Data can be a bag that contains Seeds, Transmissions and any communication between parties.

FYI, Audit Log and Transmissions docs use capital letters since the beginning.

Reply:
I'm fine with "User Id and Preferences".
I also agree to use capital letters... only for PAF keywords ;-)
Let's use that as a rule for the future!

Reply:
FYI, I already extracted the Glossary from audit requirements, and moved it at the end of "workflows" file.